packages/e2e-tests/src/specs/dashboard/integrations/sitekit.js

```diff
@@ -25,19 +25,23 @@ import { activatePlugin, deactivatePlugin } from '@wordpress/e2e-test-utils';
 import { visitSettings } from '@web-stories-wp/e2e-test-utils';
 
 describe('Site Kit integration with dashboard', () => {
-  beforeAll(async () => {
-    await activatePlugin('e2e-tests-site-kit-mock');
-  });
+  it('should be able see Site Kit specific message for Google Analytics', async () => {
+    await activatePlugin('e2e-tests-site-kit-analytics-mock');
```

Comment:
The plugin slug also needs to be updated in `packages/e2e-tests/src/specs/editor/integrations/sitekit.js` (e2e tests are currently failing because of it).

In `packages/e2e-tests/src/specs/editor/integrations/sitekit.js` we can also complement the Analytics test and add one for AdSense

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/7622/commits/d42d63c95de3d0d1ad6e02ea5edbd25737379912